Lib/fontTools/varLib/models.py

```diff
@@ -182,7 +182,7 @@ def getMasterLocationsSortKeyFunc(locations, axisOrder=[]):
 			axis = next(iter(loc))
 			value = loc[axis]
 			if axis not in axisPoints:
-				axisPoints[axis] = {0}
+				axisPoints[axis] = {0.}
```

Comment:
Should the XML writer be changed to always write float?  Not saying that this change is wrong, but might be better to also review the XML bits.